### DISCUSSION TITLE
New Tesla Energy System Alert - OpticasterExe

### DISCUSSION THREAD

**Discussion by s0schen** [jasonacox/Powerwall-Dashboard#257] [General]
Just noticed another alert to add to the list.
 OpticasterExe 

Showed up as system was transitioning from battery power to grid power for TOU adjustment at 9pm

https://www.tesla.com/en_eu/support/energy/tesla-software/opticaster

I wonder if people on the Tesla Electricity Plan see this more often when it dynamically tracks real time pricing.

![image](https://user-images.githubusercontent.com/4704443/234583707-a2a2371b-694a-42e8-9dbb-bd986f638912.png)

**Comment by jasonacox** [comment]
Nice! I added this to the list.
